src/modules/rewards/components/DashboardSection.tsx

```diff
@@ -1,96 +1,239 @@
 import { FC } from 'react';
 import { css } from 'styled-components';
-import { Box, Button, RewardsCircle, Text } from 'blocks';
+
+//Hooks
+import { useAccount } from 'hooks';
+import { useGetUserRewardsDetails } from 'queries';
+
+//components
+import { Box, Button, HoverableSVG, RewardsCircle, Refresh, Skeleton, Text } from 'blocks';
 
 export type DashboardSectionProps = {
   onGetStarted: () => void;
 };
 
 const DashboardSection: FC<DashboardSectionProps> = ({ onGetStarted }) => {
+  const { isWalletConnected, chainId, account } = useAccount();
+  let walletAddress = 'eip155:' + account;
```

Comment:
Please us a helper function for this if available ? I am sure there will be or create one.